### DISCUSSION TITLE
Discussion for standardized Initial form for Personal Loan & Invoice based loan

### DISCUSSION THREAD

**Discussion by 92shreyansh** [ONDC-Official/ONDC-FIS-Specifications#95] [Q&A]
Hi All,

Lets use this thread to standardizing Initial form i.e. step 3 of flow for Personal Loan & Invoice Based Loan.

This shall cover the standard id's for field along with its know standard values for some feilds which are drop down.

**Comment by jainish-shah-2212** [comment]
@92shreyansh  we are trying out the flow for PERSONAL_LOANS on staging env . We are getting following response as part of on_search request 


` "xinput": {
                                "head": {
                                    "descriptor": {
                                        "name": "Customer Information"
                                    },
                                    "index": {
                                        "min": 0,
                                        "cur": 0,
                                        "max": 0
                                    },
                                    "headings": [
                                        "Personal Information"
                                    ]
                                },
                                "form": {
                                    "id": "f070ac1e-4e21-493f-8220-35f37898ed0a",
                                    "url": "https://ondctest.refo.dev/loans/udform/f070ac1e-4e21-493f-8220-35f37898ed0a",
                                    "mime_type": "text/html",
                                    "resubmit": false,
                                    "multiple_sumbissions": false
                                },
                                "required": true
                            },`

As mentioned here we can have multiple forms that can be submitted using select request , where exactly do we get the other forms ? is it during subsequent on_select request , if yes then how do we determine when to trigger on_init ?

**Comment by madhus1025** [comment]
Which api's are you using for staging ?  for select , search & init ?
